src/Lucene.Net/Index/LogByteSizeMergePolicy.cs

```diff
@@ -81,17 +83,31 @@ public virtual double MaxMergeMB
 	        }
 	    }
 
-	    /// <summary>Gets or sets the minimum size for the lowest level segments.
-	    /// Any segments below this size are considered to be on
-	    /// the same level (even if they vary drastically in size)
-	    /// and will be merged whenever there are mergeFactor of
-	    /// them.  This effectively truncates the "long tail" of
-	    /// small segments that would otherwise be created into a
-	    /// single level.  If you set this too large, it could
-	    /// greatly increase the merging cost during indexing (if
-	    /// you flush many small segments). 
-	    /// </summary>
-	    public virtual double MinMergeMB
+        public virtual double LargeSegmentSizeMB
```

Comment:
Would be nice to put a comment here, explaining why do we actually want to have dedicated settings to _large_ segments.

Reply:
done